doc/source/python_tutorial.rst

```diff
@@ -26,17 +25,14 @@ What you can do with MNE Python
     - **Visualization of sensor and source space data**
     - **Time-frequency** analysis with Morlet wavelets (induced power, intertrial coherence, phase lock value) also in the source space
     - **Spectrum estimation** using multi-taper method
+    - **Mixed Source Models** combining cortical and subcortical structures
+    - **Dipole Fitting**
+    - **Decoding** multivariate pattern analyis of M/EEG topographies
     - **Compute contrasts** between conditions, between sensors, across subjects etc.
     - **Non-parametric statistics** in time, space and frequency (including cluster-level)
     - **Scripting** (batch and parallel computing)
 
-What you're not supposed to do with MNE Python
-----------------------------------------------
-
-	- **Boundary Element Modeling** use MNE and Freesurfer.
-
-
-.. note:: Package based on the FIF file format from Neuromag but can work with CTF and 4D after conversion to FIF.
```

Comment:
I think it's okay to leave this part in. It's true that people should look to FreeSurfer and MNE-C to set these things up (once per subject)